src/app/components/PlayerPanel.ts

```diff
@@ -150,85 +198,68 @@ export class PlayerPanel {
     return rollBtn;
   }
 
-  private createExchangeButton(): HTMLElement {
-    const tradeBtn = Render.elementFactory(
-      'button',
-      {
-        id: 'exchange',
-        className: 'btn button',
-      },
-      'Exchange',
-    );
-    tradeBtn.addEventListener('click', () => this.view.handleTrade());
-    return tradeBtn;
-  }
-
   /**
    * Displays results of Dice throw
    * @param diceResults Takes array of dice results
    * @param playerGain Takes array of tuples containing players gain
    */
   displayRollResult(
     diceResults: AnimalNames[],
-    playerGain: [AnimalNames, number][],
+    // playerGain: [AnimalNames, number][],
+    // player: Player,
   ): void {
     const diceResult = Render.elementFactory(
       'div',
       {},
-      Render.elementFactory(
-        'h3',
-        { className: 'player-panel__result--dice' },
-        `Dice results:`,
-      ),
       ...diceResults.map((name) =>
         ConvertAnimalName.toHTMLElement(name, 'player-panel__image'),
       ),
     );
     Render.render(
       '.player-panel__result',
       diceResult,
-      Render.elementFactory(
-        'div',
-        {},
-        Render.elementFactory(
-          'h3',
-          { className: 'player-panel__result--gain' },
-          `${this.player.theName} gains:`,
-        ),
-        ...this.convertAnimalsToHTML(
-          playerGain.map(([animal, count]) => [
-            ConvertAnimalName.toAnimalObject(animal),
-            count,
-          ]),
-        ),
-      ),
+      // Render.elementFactory(
+      // 'div',
+      // {},
+      // Render.elementFactory(
+      // 'h3',
+      // { className: 'player-panel__result--gain' },
+      // `${player.theName} gains:`,
+      // ),
+      // ...this.convertAnimalsToHTML(
+      // playerGain.map(([animal, count]) => [
+      // ConvertAnimalName.toAnimalObject(animal),
+      // count,
+      // ]),
+      // ),
+      // ),
     );
     this.view.stopTimer();
-    setTimeout(() => this.hideTimer(), 10);
+    // setTimeout(() => this.hideTimer(), 10);
   }
 
-  private hideTimer(): void {
+  /*private hideTimer(): void {
     (document.querySelector(
       '#time-left',
     ) as HTMLElement).style.display = 'none';
-  }
+  }*/
 
   /**
    * Updates timer on player panel
    * @param timeLeft accepts number value for time left
    */
-  updateTime(timeLeft: number): void {
+  /*updateTime(timeLeft: number): void {
     const timer = document.querySelector('#time-left') as HTMLElement;
     timer.innerText = `Time left: ${timeLeft} sec.`;
-  }
+  }*/
 
-  turnAlert(): void {
+  turnAlert(player: Player): void {
```

Comment:
This method is not happening anymore, the alerts should go through to the alert component. Please look if you can already send the right alert to the right place.